.gitignore

```diff
@@ -14,6 +14,7 @@ out/
 
 # Bazel artifacts
 **/bazel-*
+MODULE.bazel.lock
```

Comment:
should be part of the PR365 not here...

Reply:
Whether you use bzlmod or not, new versions of bazel will create this file.
I agree that it doesn't have to be part of this patch though so I'll commit it at head and sync.